slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -735,6 +743,31 @@ class SlideToActView @JvmOverloads constructor(
         animSet.start()
     }
 
+    /** Private method to update view to complete state immediately after view created */
+    private fun setCompleteState() {
+        mPosition = mAreaWidth - mAreaHeight
+        mActualAreaMargin = mAreaHeight / 2
+        mActualAreaWidth = mPosition / 2
+        mIsCompleted = true
+        isEnabled = false
+        val tickListener = ValueAnimator.AnimatorUpdateListener {
+            // We need to enable the drawing of the AnimatedVectorDrawable before starting it.
+            if (!mFlagDrawTick) {
+                mFlagDrawTick = true
+                mTickMargin = mIconMargin
+            }
+        }
+        val tickAnimator: ValueAnimator = createIconAnimator(this, mDrawableTick, tickListener)
+        tickAnimator.start()
```

Comment:
This should not be animated. The whole point of the `setCompleted` is to reach the completed state without animation essentially :)

Reply:
Replaced by `startIconAnimation(icon: Drawable)`. I was able to find only such a way to show `mDrawableTick`, but maybe you could advise, how to do it completely without animation.

Reply:
I think it's ok to call `startIconAnimation` in this specific section. Agree that is not fully ideal as you would want no animations at all, but this still an improvement from the status quo 👍